#ifndef IMAGETHUMB_H
#define IMAGETHUMB_H

#include <QWidget>
#include "Image/initImage.h"

#include "ui_plot_widget.h"
//-----------------------------------
// class Image Widget
#include "ui_image_widget.h"
//-----------------------------------
// class Terminal
#include "Image/image_master.h"
//-----------------------------------
// class Terminal
#include "Base/terminal.h"
//-----------------------------------

#endif //IMAGETHUMB_H


